distribution/docker/src/docker/bin/docker-entrypoint.sh

```diff
@@ -56,18 +40,13 @@ if [[ -f bin/elasticsearch-users ]]; then
   # enabled, but we have no way of knowing which node we are yet. We'll just
   # honor the variable if it's present.
   if [[ -n "$ELASTIC_PASSWORD" ]]; then
-    [[ -f /usr/share/elasticsearch/config/elasticsearch.keystore ]] || (run_as_other_user_if_needed elasticsearch-keystore create)
-    if ! (run_as_other_user_if_needed elasticsearch-keystore list | grep -q '^bootstrap.password$'); then
-      (run_as_other_user_if_needed echo "$ELASTIC_PASSWORD" | elasticsearch-keystore add -x 'bootstrap.password')
+    [[ -f /usr/share/elasticsearch/config/elasticsearch.keystore ]] || (elasticsearch-keystore create)
+    if ! (elasticsearch-keystore list | grep -q '^bootstrap.password$'); then
+      (echo "$ELASTIC_PASSWORD" | elasticsearch-keystore add -x 'bootstrap.password')
     fi
   fi
 fi
 
-if [[ "$(id -u)" == "0" ]]; then
-  # If requested and running as root, mutate the ownership of bind-mounts
-  if [[ -n "$TAKE_FILE_OWNERSHIP" ]]; then
```

Comment:
We'll need to adjust the [docs](https://github.com/elastic/elasticsearch/blob/master/docs/reference/setup/install/docker.asciidoc) and remove the `TAKE_FILE_OWNERSHIP` description as mentioned in the 3rd task in https://github.com/elastic/elasticsearch/issues/46166.

Reply:
Ah, yes, I meant to remove that from the docs. Good spot 👍